README.md

```diff
@@ -35,36 +35,6 @@ config.action_controller.asset_host = "http://my.alternative.host"
 
 to `application.rb`.
 
-## Specifying a template
```

Comment:
Not 100% sure about this removal. I agree it's more likely to be _updated_ if it's closer to the code, but I don't think you should have to read through the Slimmer code to know how to use it with a different template?